be/src/runtime/current_thread.h

```diff
@@ -4,6 +4,7 @@
 
 #include <string>
 
+#include "fmt/format.h"
```

Comment:
why need this include?

Reply:
> why need this include?

The relevant interaface is called in this header file. If it is not explicitly included, the compilation will fail.